appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/transitionmodel/BaseInterpolator.kt

```diff
@@ -83,33 +87,35 @@ abstract class BaseInterpolator<NavTarget : Any, ModelState, Props>(
                 navElement = t1.element,
                 modifier = elementProps.modifier.composed {
                     LaunchedEffect(update) {
-                        if (update.animate) {
-                            elementProps.animateTo(
-                                scope = this,
-                                props = t1.props,
-                                springSpec = currentSpringSpec,
-                                onStart = {
-                                    updateAnimationState(t1.element.id, true)
-                                },
-                                onFinished = {
-                                    updateAnimationState(t1.element.id, false)
-                                    currentSpringSpec = defaultAnimationSpec
-                                },
-                            )
-                        } else {
-                            elementProps.snapTo(this, t1.props)
+                        coroutineScope?.launch {
```

Comment:
the ? seems to be not needed

Reply:
Yeah, committed but didn't push the removal